docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:13.0-alpine
+    restart: always
+    ports:
+      - '5432:5432'
+    volumes: 
+      - db:/var/lib/postgresql/data
+      - ./db-init/init.sql:/docker-entrypoint-initdb.d/init.sql
+    env_file:
+      - ./server/.env.dev
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+ 
+
+  server:
+    build: ./server
+    image: ${IMAGE_NAME_FOR_RUN}
+    command: >
+      sh -c "
+      python ./manage.py migrate && \
+      python ./manage.py loaddata cpho/fixtures/dimension_lookups.yaml && \
+      python ./manage.py loaddata cpho/fixtures/periods.yaml && \
+      python ./manage.py runscript cpho.scripts.dev && \
+      exec gunicorn --bind 0.0.0.0:8080 --workers 1 --threads 8 --timeout 0 server.wsgi:application"
+    ports:
+      - 8080:8080
+    env_file:
+      - ./server/.env.dev
+    environment:
+      - DB_HOST=db
+      - POSTGRES_HOST_AUTH_METHOD=trust 
```

Comment:
Might be worth adding a comment here to note that values set in `environment` will effectively over-write the values in `.env.dev`, because `decouple.Config` looks in OS env vars before .env files. Important because if that behaviour ever changes, or if code running in this container reads from the `.env` file without using `decouple`, then this won't work for setting the `DB_HOST` value any more.